cypress/integration/ete-okta/registration_2.6.cy.ts

```diff
@@ -683,4 +683,329 @@ describe('Registration flow - Split 2/2', () => {
 			});
 		});
 	});
+
```

Comment:
I'll be honest, I didn't pick through these tests with a fine-tooth comb. I'm hoping that any problems will be self-revealing!